bin/critterget.py

```diff
@@ -12,14 +12,7 @@
 baseurl = "https://developers.crittercism.com:443/v1.0/"
 authbaseurl = "https://developers.crittercism.com/v1.0/"
 
-# FOR EU PoP USERS ONLY:
-# If you are using Crittercism through its EU data center presence,
-# uncomment the following two lines for baseurl and authbaseurl and 
-# comment out the previous two lines for baseurl and authbaseurl
-# baseurl = "https://developers.eu.crittercism.com:443/v1.0/"
-# authbaseurl = "https://developers.eu.crittercism.com/v1.0/“
-
-debug = 0
+debug = 1
```

Comment:
Debugging should not be enabled in our source code.  It's fine to change this when you're developing.